Project/Assets/ML-Agents/Examples/FoodCollector/Prefabs/FoodCollectorArea.prefab

```diff
@@ -2213,7 +2214,7 @@ MonoBehaviour:
   goodMaterial: {fileID: 2100000, guid: c67450f290f3e4897bc40276a619e78d, type: 2}
   frozenMaterial: {fileID: 2100000, guid: 66163cf35956a4be08e801b750c26f33, type: 2}
   myLaser: {fileID: 1081721624670010}
-  contribute: 0
+  contribute: 1
```

Comment:
None of the agents had this set, so we never accumulate the score:
https://github.com/Unity-Technologies/ml-agents/blob/2c1f04e536beb8a0d98abfa1ab288008d92d48e2/Project/Assets/ML-Agents/Examples/FoodCollector/Scripts/FoodCollectorAgent.cs#L257-L260